src/tools/perf/perftest.c

```diff
@@ -649,6 +650,14 @@ static ucs_status_t parse_test_params(ucx_perf_params_t *params, char opt, const
 #else
             ucs_error("not built with cuda support");
             return UCS_ERR_INVALID_PARAM;
+#endif
+        } else if(!strcmp(optarg, "managed")) {
+#if HAVE_CUDA
```

Comment:
can we move this to above #if HAVE_CUDA by comparing prefix in "cuda" and "cuda-managed"?

Reply:
👍 The code is a bit hard to read because of all the #ifdefs

Reply:
I've changed it now. Will push the commit soon.